modules/pergel.yml

```diff
@@ -0,0 +1,18 @@
+name: pergel
+description: >-
+  Pergel is a tailor-made solution for Nuxt and Nitro, providing swift project
+  kickstarts and seamless integration of various modules.
+repo: oku-ui/pergel
+npm: pergel-system
```

Comment:
'@pergel/nuxt'

how to change, autofix change :/

Reply:
It's picking it up from your github repo